README.md

```diff
@@ -69,41 +118,58 @@ This will stop but not delete the Airflow docker stack, for ease of restart if y
 
 #### Start Bash in Given Docker Instance
 
-Run shell in Airflow Worker instance:
+Run command shell in Airflow Worker instance:
 
 ```
  $ make tty-worker
 ```
 
-Run shell in Airflow Webserver instance:
+Run command shell in Airflow Webserver instance:
 
 ```
 $ make tty-webserver
 ```
 
-Run shell in Airflow Scheduler instance:
+Run command shell in Airflow Scheduler instance:
 
 ```
 $ make tty-scheduler
 ```
 
 ### Start Bash as Root in Given Docker Instance
 
-Run shell as root in Airflow Worker instance:
+Run command shell as root in Airflow Worker instance:
 
 ```
 $ make tty-root-worker
 ```
 
-Run shell as root in Airflow Webserver instance:
+Run command shell as root in Airflow Webserver instance:
 
 ```
 $ make tty-root-webserver
 ```
-Run shell as root in Airflow Scheduler instance:
+Run command shell as root in Airflow Scheduler instance:
 
 ```
 $ make tty-root-scheduler
 ```
```

Comment:
We recently added in both make and lint commands to the Makefile.  Can you add those here too?

Reply:
Added other Makefile targets.